src/win/util.c

```diff
@@ -1135,6 +1136,13 @@ int uv_getrusage(uv_rusage_t *uv_rusage) {
     return uv_translate_sys_error(GetLastError());
   }
 
+  ret = GetProcessMemoryInfo(GetCurrentProcess(),
+                             &memCounters,
+                             sizeof(PROCESS_MEMORY_COUNTERS));
```

Comment:
Well it's compiling and handing back the data (to my surprise). Would it be correct though? Windows documentation stated it as is above, however I also see post like [this on SO](http://stackoverflow.com/questions/8813129/practical-usage-of-virtual-memory) where they needed to cast.